src/contracts/treasury/Collector.sol

```diff
@@ -35,7 +36,7 @@ contract Collector is VersionedInitializable, ICollector, ReentrancyGuard {
   /**
    * @notice Current revision of the contract.
    */
-  uint256 public constant REVISION = 5;
```

Comment:
Better to co-locate all constants at the top of the contract.
It's weird that they are not atm and this way it's easier to see how storage slots are utilized - was looking quite a while till i found the `_fundsAdmin`.